src/views/List.jsx

```diff
@@ -25,7 +25,7 @@ export function List({ data }) {
 					<p>Welcome to {listName}!</p>
 					<p>Ready to add your first item? Start adding below!</p>
 
-					<AddItems />
+					<AddItems items={items} />
```

Comment:
based on the failed test,  i think you need to import items to use it as a prop for the AddItems component

Reply:
hey! thank you for catching that! i forgot to push the last changes, i think that this was breaking our deployed app, so i added this change